rubocop/rubocop.gemspec

```diff
@@ -6,6 +6,10 @@ Gem::Specification.new do |spec|
   # x.y.z.t
   # Where x.y.z = rubocop version
   # t is incremental
+
+  # TODO: make as a separated gem hosted via rubygems
```

Comment:
What do you mean?

Reply:
Push to the rubygems and install via it so we can simply use any versions of this gem and giving a full control over it.

Reply:
Full controll over the gem and CI with better versioning described below

Reply:
at this moment we have only github-hosted project with "commit-based" versioning :\

Reply:
> at this moment we have only github-hosted project with "commit-based" versioning :\

There is rubygems-hosted gem: https://rubygems.org/gems/rubocop-config-umbrellio.
`umbrellio` (@tycooon know who owns that account) has an access to push it

Reply:
i know. but ok :)